### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -57,4 +57,6 @@ done
 # If there are whitespace errors, print the offending file names and fail.
 git diff-index --check --cached $against --
 
+npm shrinkwrap --dev
```
Comment (danni) [inline .githooks/pre-commit]: Bad idea IMO.

File: src/components/Chevron.js [hunk 2]
```diff
@@ -1,19 +1,12 @@
...
-            <i
-                className="material-icons"
-                style={{
-                    marginLeft: "auto",
-                }}
-            >
-                {/* icon: chevron_right */}
-                &#xE5CC;
-            </i>
+            <Icon iconCode="&#xE5CC;" iconName="chevron_right" />
```
Comment (danni) [inline src/components/Chevron.js]: Can we move the codes into a hash?

File: .githooks/pre-commit (hunk 1 above)
Comment (DanielHeath) [inline .githooks/pre-commit]: Could you elaborate? 

I usually prefer to know which version of my dependencies I'm going to run.

File: src/components/Chevron.js (hunk 2 above)
Comment (DanielHeath) [inline src/components/Chevron.js]: Good idea.

File: .githooks/pre-commit (hunk 1 above)
Comment (danni) [inline .githooks/pre-commit]: You should update the deps manually. Not as part of the commit hook. Otherwise it can lead to surprises and/or insanity.

Comment (DanielHeath) [inline .githooks/pre-commit]: Shrinkwrap doesn't update versions of things - it records what you currently have installed, and fails if you have extra stuff installed or missing packages.

Comment (DanielHeath) [inline .githooks/pre-commit]: More generally, I'd like to fail the commit if you add a new dependency and don't shrinkwrap it. Not actually fussed as to how we do that.

Comment (danni) [inline .githooks/pre-commit]: Oh, what writes out the JSON file then? I might be confused by the tools. I'm more than happy for something that fails if you haven't updated package.json.

**Comment by danni** [comment]
++

**Pull request merged**